lib/beaker/subcommand.rb

```diff
@@ -53,18 +53,15 @@ def initialize(*args)
     class_option :'exclude-tags', :type => :string, :group => 'Beaker run'
     class_option :'xml-time-order', :type => :boolean, :group => 'Beaker run'
 
-    #global class options that should be available for all beaker subcommands
```

Comment:
Is there a reason why this global class_option is removed and then added individually per subcommand?